src/routes/login/+page.svelte

```diff
@@ -1,22 +1,115 @@
 <script>
+    // Framework imports
+    import { getToastStore } from "@skeletonlabs/skeleton";
+
+    // Import 🐦
+    import { ConstantBackoff, handleAll, retry } from "cockatiel";
+
+    // Import backend urls
+    import { urls } from "$lib/util.js";
+
     // Definitions
-    let somethingWrong = false;
+    // Constants
+    const toastStore = getToastStore();
+
+    const mailRegexPattern =
+        /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+    // Enums
+    const btnIcon = {
+        locked: "🔐",
+        unlocked: "🔓",
+    };
 
     const indicatorStatus = {
-        none: "",
+        none: "border-surface-700",
         sucess: "input-success",
-        warning: "input-warning",
-        error: "input-error",
+        warning: "border-warning-400",
+        error: "border-error-600",
```

Comment:
Durfte die suggestion nicht in einem machen.
```suggestion
        warning: "!border-warning-400",
        error: "!border-error-600",
```